preprocessing/nextclade/src/loculus_preprocessing/prepro.py

```diff
@@ -728,7 +734,7 @@ def process_single(  # noqa: C901
         version=version_from_str(id),
         data=ProcessedData(
             metadata=output_metadata,
-            unalignedNucleotideSequences=unprocessed.unalignedNucleotideSequences,
+            unalignedNucleotideSequences=unaligned_nucleotide_sequences,
```

Comment:
We need to trim this before the unaligned sequence is sent to nextclade - as it is we send the sequences with Ns to nextclade and this is reflected in the listed insertions and `Total unknown nucs` numbers - these then no longer match with the unaligned sequence shown on the webpage. 

I took the liberty of making a PR for this: https://github.com/loculus-project/loculus/pull/4079

Reply:
Thank you!